build.gradle

```diff
@@ -109,12 +110,48 @@ task sourcesJar(type: Jar) {
     from sourceSets.main.allJava
 }
 
+task javadocJar(type: Jar) {
+   classifier = 'javadoc'
+   from javadoc.destinationDir
+}
+
 publishing {
     publications {
         shadow(MavenPublication) { publication ->
             project.shadow.component(publication)
             artifact sourcesJar
         }
+        sonatype(MavenPublication) { publication ->
+            artifactId = "opendistro_sql_jdbc"
```

Comment:
remove this - artifact id is picked automatically from settings.gradle

Reply:
Ok. I will remove it.